agent/statemanager/state_manager.go

```diff
@@ -115,7 +115,7 @@ const (
 	//	 a) Add 'authorizationConfig', 'transitEncryption' and 'transitEncryptionPort' to 'taskresource.volume.EFSVolumeConfig'
 	//	 b) Add 'pauseContainerPID' field to 'taskresource.volume.VolumeResource'
 	// 28) Add 'envfile' field to 'resources'
-	// 29) Add 'ExecCommandAgentMetadata' field to 'apicontainer.Container'
+	// 29) Add 'ManagedAgentsUnsafe' field to 'apicontainer.Container'
```

Comment:
looks like this is not needed anymore with boltdb migration. lets confirm with @fenxiong

Reply:
ok, can we remove it later if we confirm is not needed?